ocaml-eglot-util.el

```diff
@@ -99,5 +99,17 @@
       (list :start start
             :end (ocaml-eglot-util--position-increase-char start "_")))))
 
+(defun ocaml-eglot-util--visit-file (strategy current-file new-file range)
+  "Visits a referenced document, NEW-FILE at position  start of RANGE.
+The STRATEGY can be `'new' `'current' or `'smart'.  The later opens a
+new buffer if the destination is not in the CURRENT-FILE, ans uses the
+current buffer otherwise."
```

Comment:
Reading the readme I now see that the correct notion here is not buffer but window, isn't it ? 

```suggestion
(defun ocaml-eglot-util--visit-file (strategy current-file new-file range)
  "Visits a referenced document, NEW-FILE at position  start of RANGE.
The STRATEGY can be `'new' `'current' or `'smart'.  The later opens a
new window if the destination is not in the CURRENT-FILE, ans uses the
current window otherwise."
```